### PR TITLE
Update multizone.rst

### PR DESCRIPTION
**Pull request by P1532**

### CODE REVIEW

File: doc/source/user_guide/multizone.rst [hunk 1]
```diff
@@ -6,16 +6,16 @@ MultiZone controls
 
 *This is a Beta feature. API Behavior and Implementation  may change in future.*
```
Comment (PipKat) [inline doc/source/user_guide/multizone.rst:7]:
```suggestion
*This is a beta feature. API behavior and implementation might change in the future.*
```

File: doc/source/user_guide/multizone.rst [hunk 2]
```diff
@@ -6,16 +6,16 @@ MultiZone controls
...
 
-:class:`MultiZoneControl <ansys.meshing.prime.MultiZoneControl>` class provides automatic decomposition of geometry into mapped (sweepable) regions and free regions. 
+The :class:`MultiZoneControl <ansys.meshing.prime.MultiZoneControl>` class provides automatic geometry decomposition into mapped (sweepable) regions and free regions. 
 Mapped (sweepable) regions are filled with hexahedral elements and free regions are filled with non-hexahedral elements.
 When you perform MultiZone Method meshing, all regions are meshed with a pure hexahedral mesh if possible.  
 
-:class:`AutoMesh <ansys.meshing.prime.automesh>` class enables you to automatically create the hex mesh on the scoped bodies using multizone meshing algorithms. 
+The :class:`AutoMesh <ansys.meshing.prime.automesh>` class enables you to automatically create the hex mesh on the scoped bodies using multizone meshing algorithms.
```
Comment (PipKat) [inline doc/source/user_guide/multizone.rst:13]:
```suggestion
The :class:`AutoMesh <ansys.meshing.prime.automesh>` class lets
 you automatically create the hex mesh on the scoped bodies using multizone meshing algorithms. 
```

File: doc/source/user_guide/multizone.rst [hunk 3]
```diff
@@ -6,16 +6,16 @@ MultiZone controls
...
 *This is a Beta feature. API Behavior and Implementation  may change in future.*
 
-:class:`MultiZoneControl <ansys.meshing.prime.MultiZoneControl>` class provides automatic decomposition of geometry into mapped (sweepable) regions and free regions. 
+The :class:`MultiZoneControl <ansys.meshing.prime.MultiZoneControl>` class provides automatic geometry decomposition into mapped (sweepable) regions and free regions. 
 Mapped (sweepable) regions are filled with hexahedral elements and free regions are filled with non-hexahedral elements.
 When you perform MultiZone Method meshing, all regions are meshed with a pure hexahedral mesh if possible.  
 
-:class:`AutoMesh <ansys.meshing.prime.automesh>` class enables you to automatically create the hex mesh on the scoped bodies using multizone meshing algorithms. 
+The :class:`AutoMesh <ansys.meshing.prime.automesh>` class enables you to automatically create the hex mesh on the scoped bodies using multizone meshing algorithms. 
 :func:`AutoMesh.mesh() <ansys.meshing.prime.AutoMesh.mesh>` method allows you to perform MultiZone meshing with given MultiZone control.
```
Comment (PipKat) [inline doc/source/user_guide/multizone.rst:14]:
```suggestion
The :func:`AutoMesh.mesh() <ansys.meshing.prime.AutoMesh.mesh>` function lets you perform multizone meshing with the given multizone control. 
```

File: doc/source/user_guide/multizone.rst [hunk 4]
```diff
@@ -6,16 +6,16 @@ MultiZone controls
...
-:class:`MultiZoneControl <ansys.meshing.prime.MultiZoneControl>` class provides automatic decomposition of geometry into mapped (sweepable) regions and free regions. 
+The :class:`MultiZoneControl <ansys.meshing.prime.MultiZoneControl>` class provides automatic geometry decomposition into mapped (sweepable) regions and free regions. 
 Mapped (sweepable) regions are filled with hexahedral elements and free regions are filled with non-hexahedral elements.
 When you perform MultiZone Method meshing, all regions are meshed with a pure hexahedral mesh if possible.  
 
-:class:`AutoMesh <ansys.meshing.prime.automesh>` class enables you to automatically create the hex mesh on the scoped bodies using multizone meshing algorithms. 
+The :class:`AutoMesh <ansys.meshing.prime.automesh>` class enables you to automatically create the hex mesh on the scoped bodies using multizone meshing algorithms. 
 :func:`AutoMesh.mesh() <ansys.meshing.prime.AutoMesh.mesh>` method allows you to perform MultiZone meshing with given MultiZone control. 
 
 The below example shows how MultiZone control can be applied on a body:
```
Comment (PipKat) [inline doc/source/user_guide/multizone.rst:16]:
```suggestion
The following example shows to apply multizone controls on a body: 
```

File: doc/source/user_guide/multizone.rst [hunk 5]
```diff
@@ -6,16 +6,16 @@ MultiZone controls
...
 When you perform MultiZone Method meshing, all regions are meshed with a pure hexahedral mesh if possible.  
 
-:class:`AutoMesh <ansys.meshing.prime.automesh>` class enables you to automatically create the hex mesh on the scoped bodies using multizone meshing algorithms. 
+The :class:`AutoMesh <ansys.meshing.prime.automesh>` class enables you to automatically create the hex mesh on the scoped bodies using multizone meshing algorithms. 
 :func:`AutoMesh.mesh() <ansys.meshing.prime.AutoMesh.mesh>` method allows you to perform MultiZone meshing with given MultiZone control. 
 
 The below example shows how MultiZone control can be applied on a body: 
 
-1. Read the model.
+1. Start the PyPrimeMesh client and read the model. The model is made up of two topo volumes that share a connected topo face between them. The two topo volume have volume zones defined such as left volume zone is **solid1** and right is **solid**.
```
Comment (PipKat) [inline doc/source/user_guide/multizone.rst:18]:
```suggestion
1. Start the PyPrimeMesh client and read the model. The model is made up of two topo volumes that share a connected topo face between them. The two topo volumes have volume zones defined. The left volume zone is ``solid1``, and the right volume zone is ``solid``.
```
Use double backticks for the names as they are code entities.

File: doc/source/user_guide/multizone.rst [hunk 6]
```diff
@@ -62,12 +62,14 @@ The below example shows how MultiZone control can be applied on a body:
 
    multizone_control = model.control_data.create_multi_zone_control()
 
+3. Define the volume scope and surface scope within the model and applying the volume scope and surface scope to the Multizone Control. In this example, volume scope is scoped specifically to "solid1" to show the difference between the MultiZone mesh and automesh
```
Comment (PipKat) [inline doc/source/user_guide/multizone.rst:65]:
```suggestion
3. Define the volume scope and surface scope within the model and apply the volume scope and surface scope to the multizone control.
   In this example, the volume scope is scoped specifically to ``solid1`` to show the difference between the multizone mesh and automesh.
```

File: doc/source/user_guide/multizone.rst [hunk 7]
```diff
@@ -62,12 +62,14 @@ The below example shows how MultiZone control can be applied on a body:
...
 
-3. Define the volume scope and surface scope for the model to apply MultiZone control on the same.
+.. note::
+  Keep the surface scope as the complete geometry (*)
```
Comment (PipKat) [inline doc/source/user_guide/multizone.rst:68]:
```suggestion
  Keep the surface scope as the complete geometry (*).
```I don't understand the significance of the "(*)" here. Perhaps additional clarification is needed?

Comment (P1532) [reply doc/source/user_guide/multizone.rst:68]: @PipKat For complete geometry we are adding it as *

File: doc/source/user_guide/multizone.rst [hunk 8]
```diff
@@ -6,16 +6,16 @@ MultiZone controls
 
 *This is a Beta feature. API Behavior and Implementation  may change in future.*
 
-:class:`MultiZoneControl <ansys.meshing.prime.MultiZoneControl>` class provides automatic decomposition of geometry into mapped (sweepable) regions and free regions. 
+The :class:`MultiZoneControl <ansys.meshing.prime.MultiZoneControl>` class provides automatic geometry decomposition into mapped (sweepable) regions and free regions. 
 Mapped (sweepable) regions are filled with hexahedral elements and free regions are filled with non-hexahedral elements.
 When you perform MultiZone Method meshing, all regions are meshed with a pure hexahedral mesh if possible.
```
[the remark this answers is not in the document]
Comment (P1532) [reply doc/source/user_guide/multizone.rst:11]: @PipKat We are Write MultiZone like these in the documents, So as of now I'm keeping the word as MultiZone and resolving these conversation

**Review by waltersma — APPROVED** [review]

**Review by waltersma — APPROVED** [review]

**Pull request merged**